### ISSUE TITLE
Communicate with Brain and Raspi

### ISSUE THREAD

**Issue by schromya** [UAA-Robo/VEX-U-Over-Under#21]
### Description
We need to send data between the Brain and the Raspberry Pi. You probably need to run a python program on the Raspberry Pi to do this. 

Resources:
(forum)[https://www.vexforum.com/t/rapsberry-pi-to-vex-bidirectional-communication/85696]

<br> 

### DOD
This task its considered done when it is thoroughly documented in the [Engineering Notebook](https://docs.google.com/document/d/1auSQpJCPH_xxbiL1h6w8SKAvTJxroBNW1Kcgkotj7ro/edit?usp=sharing).

**Comment by pixama** [comment]
I found some code using serial where you find a serial port "/dev/ttyUBS*/" but that command line didn't pull up any of the ports needed or referenced. So I used "lsusb" and found which Rasb Pi port was connected to the Vex controller but it only showed the BUS Device. I don't know how I would write a program for that :)

 - help.

**Comment by TooSquishie** [comment]
https://www.vexforum.com/t/reading-vex-v5-serial-port-using-python/84699

Might be useful

**Comment by pixama** [comment]
Yeah, I just checked that link. The problem for me was.. I couldn't find the ACM0/ACM1 ports. So I had no idea which one the vex controller was/ or could be connected to.
